cmake/libs/Libbacktrace.cmake

```diff
@@ -21,7 +21,16 @@ ExternalProject_Add(project_libbacktrace
   SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/../../3rdparty/libbacktrace
   BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/libbacktrace
   CONFIGURE_COMMAND "${CMAKE_CURRENT_LIST_DIR}/../../3rdparty/libbacktrace/configure"
-                    "--prefix=${CMAKE_CURRENT_BINARY_DIR}/libbacktrace" --with-pic
+                    "--prefix=${CMAKE_CURRENT_BINARY_DIR}/libbacktrace"
+                    --with-pic
+                    "CC=${CMAKE_C_COMPILER}"
+                    "CFLAGS=${CMAKE_C_FLAGS}"
+                    "LDFLAGS=${CMAKE_EXE_LINKER_FLAGS}"
+                    "CPP=${CMAKE_C_COMPILER} -E"
+                    "NM=${CMAKE_NM}"
+                    "STRIP=${CMAKE_STRIP}"
+                    "CPPFLAGS=${CMAKE_C_FLAGS}"
+                    "--host=${MACHINE_NAME}"
```

Comment:
I don't see this one documented in cmake, do you need to supply it with `cmake -DMACHINE_NAME=`?

Reply:
yes, it should be specified when executing cmake

![image](https://user-images.githubusercontent.com/3264637/116225445-1ff15a80-a752-11eb-9ae5-fac0eac7236b.png)

Reply:
actually i am not sure is ``-DCMAKE_HOST`` is neccessary, but it does not hurt (i guess)

Reply:
okay cool. would it be possible for you to add the steps you took to cross-compile to the [install from source](https://tvm.apache.org/docs/install/from_source.html) docs?

Reply:
yes, working on it, i will push in a few sec.

Reply:
done, i built the doc locally and it is rendered ok